frame/collective/src/lib.rs

```diff
@@ -1107,6 +1107,8 @@ impl<T: Config<I>, I: 'static> InitializeMembers<T::AccountId> for Pallet<T, I>
 	fn initialize_members(members: &[T::AccountId]) {
 		if !members.is_empty() {
 			assert!(<Members<T, I>>::get().is_empty(), "Members are already initialized!");
+			let mut members = members.to_vec();
+			members.sort();
```

Comment:
I guess duplicates are fine?  
Please add a duplicate to the test to check it.

Reply:
I think we should also assert the `MaxMembers` limit here.
I wonder why the `Vec` is not bounded for the `Members` storage. we should probably migrate to bounded?

Reply:
Duplicates are already tested.

MaxMembers was not - good suggestion @muharem